libr/bin/format/elf/elf.c

```diff
@@ -2625,10 +2612,10 @@ static bool sectionIsValid(ELFOBJ *bin, RBinElfSection *sect) {
 static Elf_(Xword) get_section_mode(ELFOBJ *bin, size_t pos) {
 	if (r_str_startswith (bin->g_sections[pos].name, ".rela.")) {
 		return DT_RELA;
-	} else if (r_str_startswith (bin->g_sections[pos].name, ".rel.")) {
+	}
```

Comment:
I fail to understand why this style :( although the `return` means nothing can be executed after the `if`, the `else if` makes it very clear than it is all a single "logical block". (it is also more compact, but this is a very relative benefit)

Reply:
it makes clear that there's nothing after that. the else is dead code, and visually it looks longer but cleaner